### PR TITLE
adding ux and forms to page

### PR DESCRIPTION
**Pull request by alexander16108**

### CODE REVIEW

**Review by layzrd — COMMENTED** [review]
Status: Invalid  ### ↩️
Hi @alexander16108, I am [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏 You did well on this project.
I appreciate your efforts to achieve this milestone 🦍. Kindly fix the following issues.

[REQUIRED]

In this project, 
- [ ] Fix the README file. In addition to the project title and description, you should add technologies and techniques used to build it, the authors, setup instructions, project features, etc. Kindly use the following template because it has useful sections and looks professional.
[https://github.com/microverseinc/readme-template](https://github.com/microverseinc/readme-template)
- [ ] The Pull Request should have a meaningful title and a brief description such as details about the project technologies and techniques used to build it, the authors, the tasks you accomplished, etc.). Please check the following link.
[Professional GitHub repository](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Connect me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
### Invalid submission
- [ ] Please use the same PR you submitted for the first review-request.

- [ ] Please create a Pull Request from the branch you have the actual code to the main. 

- [ ] Please don't copy-paste your readme in PR message. Write a descriptive message in PR which describes your work in the project. 

Regards,
Arslan Bsharat
@arslanbisharat

**Review by Temmarie — CHANGES_REQUESTED** [review]
### Status: Invalid!! ❗ 
Hi Team ! 👋 👋 
### Your project has been marked as invalid. Due to the following reasons:

- [ ] Please use the same PR you submitted for the first review-request.
Kindly work on them and submit them for another code review.

- [ ] Your project cannot be approved until you write a descriptive PR Message. It's essential that your repository is professional, as this is one of the things that potential colleagues will look at when recruiting team members. Please refer to [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more information.

- [ ] Please create a feature branch and open a pull request from the feature branch to the main branch.


__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Review by layzrd — COMMENTED** [review]
### Status: Require changes 🌳 ♲
Hi @alexander16108, I am [Abdulaziz Redi Ali](https://azizali.ml/), your TSE for this PR.  👏 You did well on this project.
I appreciate your efforts to achieve this milestone 🦍. Kindly fix the following issues.

[REQUIRED]

In this project, 
- [ ] Fix the README file. In addition to the project title and description, you should add technologies and techniques used to build it, the authors, setup instructions, project features, etc. Kindly use the following template because it has useful sections and looks professional.
[https://github.com/microverseinc/readme-template](https://github.com/microverseinc/readme-template)
- [ ] The Pull Request should have a meaningful title and a brief description such as details about the project technologies and techniques used to build it, the authors, the tasks you accomplished, etc.). Please check the following link.
[Professional GitHub repository](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/)
You should follow the GitHub flow.
**TL/DR**
- create a feature branch with a meaningful name 
	Example: `git checkout -b homepage`
	the `homepage` is the feature branch name
- work on it, 
- when you finish working, create a pull request from the feature (e.g. `homepage`) branch to the main branch.
 _For more check the following link_ [https://guides.github.com/introduction/flow/](https://guides.github.com/introduction/flow/)

Once you fix the issues, kindly submit another code review. ♲

Happy coding 👨‍💻
[Abdulaziz Redi Ali](https://azizali.ml/) | [Connect me on Slack](https://microverse-students.slack.com/team/U011SD6JUH2)
